addons/web/controllers/main.py

```diff
@@ -45,7 +45,6 @@
 from odoo.exceptions import AccessError, UserError, AccessDenied
 from odoo.models import check_method_name
 from odoo.service import db, security
-from odoo.addons.base.models.ir_actions import ALLOWED_IR_ACTIONS_FIELDS
```

Comment:
There should probably be some squashing / cleanup here e.g. "web: /web/action/load as public controller" and this should be merged no?